### PR TITLE
update to node-gradle plugin 3.0.0

### PR DESCRIPTION
**Pull request by cdietrich**
[WIP] update to node-gradle plugin 3.0.0
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
waits for the final release

File: org.eclipse.xtext.web/build.gradle [hunk 1]
```diff
@@ -47,29 +47,32 @@ task copyImages(type: Copy) {
...
 processResources.dependsOn(copyImages)
 
-class OptimizationTask extends com.moowork.gradle.node.task.NodeTask {
+class OptimizationTask extends com.github.gradle.node.task.NodeTask {
 	OptimizationTask() {
 		dependsOn(project.npmInstall)
 		inputs.dir 'src/main/js'
-		script = project.file('node_modules/requirejs/bin/r.js')
+		script.set(project.file('node_modules/requirejs/bin/r.js'))
 		def optimizeLogLevel = 2 // WARN
```
Comment (szarnekow) [inline org.eclipse.xtext.web/build.gradle:55]: rename `optimizeLogLevel -> logLevelWarn`

**Review by szarnekow — APPROVED** [review]

**Comment by cdietrich** [comment]
move to 2.25

**Comment by cdietrich** [comment]
we still wait for the final release of the plugin

File: org.eclipse.xtext.web/build.gradle [hunk 2]
```diff
@@ -47,29 +47,32 @@ task copyImages(type: Copy) {
 }
 processResources.dependsOn(copyImages)
 
-class OptimizationTask extends com.moowork.gradle.node.task.NodeTask {
```
Comment (cdietrich) [inline org.eclipse.xtext.web/build.gradle:50]: https://github.com/node-gradle/gradle-node-plugin/issues/137

**Pull request merged**